openjdk-21/debian/changelog

```diff
@@ -1,3 +1,12 @@
+openjdk-21 (21.0.1+12-1) unstable; urgency=high
```

Comment:
```suggestion
openjdk-21 (21.0.1+12-1) focal; urgency=high
```

Reply:
Fixed in https://github.com/coffeelibs/ppa-openjdk/pull/1/commits/6e882c643fda5069a9618c74d3e41f64cea3fa84